themes/default/content/docs/pulumi-cloud/webhooks.md

```diff
@@ -128,21 +128,23 @@ class PulumiServiceWebhook: Stack
 
 1. Navigate to **Settings** > **Webhooks**.
 2. Select **Create webhook**.
-3. Select between a slack-formatted or generic JSON webhooks.
+3. Select between a Slack, Microsoft Teams, or generic JSON webhooks.
```

Comment:
This new syntax feels off to me with dropping the `-formatted`?  But words 🤷‍♀️ - will leave it up to you.

Reply:
The original text seems a bit off too -- "select between ... a webhooks". 😄 

I can't submit the suggestion directly because GitHub, but if it helps, here's what I'd suggest:

```
1. Navigate to **Settings** > **Webhooks**.
2. Select **Create webhook**.
3. Under Destination, choose **Webhook**, **Slack**, or **Microsoft Teams**.
    * For generic JSON webhooks, provide a display name, payload URL, and optional validation secret.
    * For Microsoft Teams webhooks, provide a display name and Microsoft Teams webhook URL.
	* For Slack webhooks, provide a display name and Slack webhook URL.
4. Choose between receiving all events or only specific events using the filters menu.
```

If this looks okay to you also, would you mind applying to the Stack Webhook section also (and adjust accordingly for Deployments)?